_clients/index.md

```diff
@@ -35,14 +35,19 @@ OpenSearch provides clients for the following programming languages and platform
 * **Rust**
   * [OpenSearch Rust client]({{site.url}}{{site.baseurl}}/clients/rust/)
 
-All clients are compatible with any version of OpenSearch.
+All clients are compatible with any version of OpenSearch. However, major versions of OpenSearch introduce breaking changes that require careful upgrades of the clients. For example, while clients 2.0.0 work against the latest OpenSearch 1.x, certain deprecated features removed in OpenSearch 2.0 have also been removed from the clients. For a client compatibility matrix, see the COMPATIBILITY.md file in the client's repository.
```

Comment:
I don't think client v2 removed server 1.x functionality. Let me check on that.

Reply:
I think this statement looks correct though instead of generalizing, I think it would be best to point the users to refer the COMPATIBILITY.md for each client since it provides the accurate information for each client.